### ISSUE TITLE
0.2.0

### ISSUE THREAD

**Issue by jer78** [SphtKr/homebridge-itunes#8]
Is the new version ready to test yet? Also, where does the media info such as the album art display?

**Comment by SphtKr** [comment]
No, got stuck on some bad behavior and haven't had time to get back to it.

At present, no client supports the album art characteristic. Hesperus _might_ add it, but the author won't be getting back to working on that app until after the summer, at least.

**Comment by SphtKr** [comment]
This stuff is finally coming... check out https://github.com/SphtKr/homebridge-itunes/pull/25
